### PR TITLE
{tools}[GCCcore/6.4.0] util-linux v2.32

### PR DESCRIPTION
**Pull request by JackPerdue**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/u/util-linux/util-linux-2.32-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+source_urls = ['%s/v%%(version_major_minor)s' % homepage]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['ce43afee3182f1bddb0be83f68bd378770efb9b6fdd8f464333ff8e07903db56']
+
+# disable account related utilities (they need OS dependant pam-devel files)
+# disable wall and friends (requires group changing permissions for install user)
+# install systemd service files in install dir
+# install bash completion files in install dir
+configopts = "--disable-chfn-chsh --disable-login --disable-su "
```
Comment (boegel) [inline easybuild/easyconfigs/u/util-linux/util-linux-2.32-GCCcore-6.4.0.eb]: please also include `--disable-rfkill` here, cfr. #5480

File: easybuild/easyconfigs/u/util-linux/util-linux-2.32-GCCcore-6.4.0.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# disable building Python bindings (since we don't include Python as a dep)
+configopts += "--without-python "
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('LibUUID', '1.0.3'),
+    ('ncurses', '6.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/u/util-linux/util-linux-2.32-GCCcore-6.4.0.eb]: missing `zlib` dependency?

**Comment by migueldiascosta** [comment]
@JackPerdue, this PR is being closed for the following reason(s): closing and reopening to trigger tests.

**Comment by boegel** [comment]
This same easyconfig is already merged via #6237

Only difference here is that `LibUUID` is a dependency, which doesn't make much sense to me (see #8258)

**Pull request closed without merging**